djangogirls/urls.py

```diff
@@ -12,5 +12,6 @@
     # Regular links:
     url(r'^admin/', include(admin.site.urls)),
     url(r'^pages/', include('django.contrib.flatpages.urls')),
+    url(r'^jobs/?', include('jobs.urls', namespace='jobs')),
```

Comment:
The ? matches /jobs or /jobs/, however it means that there are two urls for the same content (even if they display the same thing), which could cause us problems.
In settings.py, add a line
```APPEND_SLASH = True```
Then, django will try to match the url, if it doesn't find anything it will append a slash and try that.
You will also need to move the jobs url line to before the pages url (otherwise anything after jobs/ - e.g. jobs/add will redirect to /events)